easybuild/easyblocks/g/gamess_us.py

```diff
@@ -33,24 +33,35 @@
 @author: Toon Willems (Ghent University)
 @author: Pablo Escobar (sciCORE, SIB, University of Basel)
 @author: Benjamin Roberts (The University of Auckland)
+@author: Alex Domingo (Vrije Universiteit Brussel)
 """
 import fileinput
 import glob
 import os
 import re
-import shutil
 import sys
 import tempfile
 
 from easybuild.framework.easyblock import EasyBlock
 from easybuild.framework.easyconfig import CUSTOM, MANDATORY
 from easybuild.tools.build_log import EasyBuildError
 from easybuild.tools.config import build_option
-from easybuild.tools.filetools import read_file, write_file
+from easybuild.tools.filetools import change_dir, copy_file, mkdir, read_file, write_file, remove_dir
 from easybuild.tools.modules import get_software_root, get_software_version
-from easybuild.tools.run import run_cmd, run_cmd_qa
-from easybuild.tools.systemtools import get_platform_name
-from easybuild.tools import toolchain
+from easybuild.tools.run import run_cmd
+from easybuild.tools.systemtools import POWER, X86_64
+from easybuild.tools.systemtools import get_cpu_architecture
+from easybuild.tools import LooseVersion, toolchain
+
+GAMESS_INSTALL_INFO = 'install.info'
+GAMESS_MPI_TEST_BLACKLIST = [
```

Comment:
maybe rename this to `GAMESS_SERIAL_TESTS` ?

Reply:
sure, fixed in https://github.com/easybuilders/easybuild-easyblocks/pull/3047/commits/c79e6e250d03ef6013b7d63677bfd25dd8f65b83 alongside a bit of refactor to run all tests in serial for non-mpi builds